README.md

```diff
@@ -0,0 +1,140 @@
+# Codex surveys
+
+A simple constructor for the feedback widget.
+
+## How to use Surveys
+
+### Basics
+
+We have 2 global functions to use the widget:
+
+```javascript
+window.feedback = {
```

Comment:
I suggest using term `survey` instead of `feedback`, as it's already used in the name of the project